engines/ultima/ultima8/metaengine.cpp

```diff
@@ -41,89 +41,89 @@ struct KeybindingRecord {
+	{ ACTION_KEYRING, "KEYRING", _s("Use keyring"), "k", "JOY_RIGHT_STICK" },
+	{ ACTION_MINIMAP, "MINIMAP", _s("Open minimap"), "m", "JOY_BACK" },
+	{ ACTION_RECALL, "RECALL", _s("Use recall"), "r", nullptr },
+	{ ACTION_INVENTORY, "INVENTORY", _s("Open inventory & statistics"), "z", "JOY_LEFT_STICK" },
 	// I18N: gump is Graphical User Menu Pop-up
-	{ ACTION_CLOSE_GUMPS, "CLOSE_GUMPS", _s("Close All Displays"), "BACKSPACE", nullptr },
+	{ ACTION_CLOSE_GUMPS, "CLOSE_GUMPS", _s("Close all displays"), "BACKSPACE", nullptr },
 	{ ACTION_JUMP, "JUMP", _s("Jump (fake both-button-click)"), "SPACE", nullptr },
-	{ ACTION_MOVE_STEP, "MOVE_STEP", _s("Careful Step"), "LCTRL", "JOY_LEFT_TRIGGER" },
+	{ ACTION_MOVE_STEP, "MOVE_STEP", _s("Careful step"), "LCTRL", "JOY_LEFT_TRIGGER" },
 
 	{ ACTION_NONE, nullptr, nullptr, nullptr, nullptr }
 };
 
 static const KeybindingRecord CRUSADER_KEYS[] = {
-	{ ACTION_NEXT_WEAPON, "NEXT_WEAPON", _s("Cycle through Weapon Inventory"), "w", "KP_MULTIPLY" },
-	{ ACTION_NEXT_INVENTORY, "NEXT_INVENTORY", _s("Cycle through Item Inventory"), "i", "KP_MINUS" },
-	{ ACTION_USE_INVENTORY, "USE_INVENTORY", _s("Use Inventory Item"), "u", "KP_PERIOD" },
-	{ ACTION_USE_MEDIKIT, "USE_MEDIKIT", _s("Use Medikit"), "m", nullptr },
-	{ ACTION_USE_ENERGYCUBE, "USE_ENERGYCUBE", _s("Use Energy Cube"), "e", nullptr },
-	{ ACTION_DETONATE_BOMB, "DETONATE_BOMB", _s("Detonate Bomb"), "b", nullptr },
+	{ ACTION_NEXT_WEAPON, "NEXT_WEAPON", _s("Cycle through weapon inventory"), "w", "KP_MULTIPLY" },
+	{ ACTION_NEXT_INVENTORY, "NEXT_INVENTORY", _s("Cycle through item inventory"), "i", "KP_MINUS" },
+	{ ACTION_USE_INVENTORY, "USE_INVENTORY", _s("Use inventory item"), "u", "KP_PERIOD" },
+	{ ACTION_USE_MEDIKIT, "USE_MEDIKIT", _s("Use medikit"), "m", nullptr },
+	{ ACTION_USE_ENERGYCUBE, "USE_ENERGYCUBE", _s("Use energy cube"), "e", nullptr },
+	{ ACTION_DETONATE_BOMB, "DETONATE_BOMB", _s("Detonate bomb"), "b", nullptr },
 	// TODO: The same key should be "use datalink" in no regret.
-	{ ACTION_DROP_WEAPON, "DROP_WEAPON", _s("Drop Weapon"), "C+d", nullptr },
-	{ ACTION_SELECT_ITEMS, "SELECT_ITEM", _s("Search / Select Item"), "s", "KP_PLUS" },
-	{ ACTION_USE_SELECTION, "USE_SELECTION", _s("Use Selection"), "RETURN", "KP_ENTER" },
-	{ ACTION_GRAB_ITEMS, "GRAB_ITEM", _s("Grab Items"), "g", nullptr },
-	{ ACTION_ATTACK, "ATTACK", _s("Fire Weapon"), "SPACE", "KP0" },
-	{ ACTION_CAMERA_AVATAR, "CAMERA_AVATAR", _s("Center screen on Silencer"), "z", nullptr },
+	{ ACTION_DROP_WEAPON, "DROP_WEAPON", _s("Drop weapon"), "C+d", nullptr },
+	{ ACTION_SELECT_ITEMS, "SELECT_ITEM", _s("Search / Select item"), "s", "KP_PLUS" },
+	{ ACTION_USE_SELECTION, "USE_SELECTION", _s("Use selection"), "RETURN", "KP_ENTER" },
+	{ ACTION_GRAB_ITEMS, "GRAB_ITEM", _s("Grab items"), "g", nullptr },
+	{ ACTION_ATTACK, "ATTACK", _s("Fire weapon"), "SPACE", "KP0" },
+	{ ACTION_CAMERA_AVATAR, "CAMERA_AVATAR", _s("Center screen on silencer"), "z", nullptr },
```

Comment:
"Silencer" or "The Silencer" is the term for the player character in Crusader. It should probably remain capitalized.
"Avatar" or "The Avatar" is similar in the Ultima games.